data/human/transport missions.txt

```diff
@@ -3209,3 +3209,92 @@ mission "Earth Retirement"
 			label reject
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
+
+
+
+mission "Spacediving professionals"
+	name "Spacediving at <planet>"
+	description "Head to destination with <bunks> skydivers, then drop them from the edge of space as you land. They are wearing thermal spacesuits, durable parachutes and oxygen bottles to ensure their survival. Payment is <payment>. This type of job has risks of non-payment and fines."
+	passengers 5 10
+	repeat
+	job
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	destination
+		distance 1 7
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	to offer
+		random < 14
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 8000
+		dialog "As you descend through the upper atmosphere, the skydivers pay you <payment> before jumping from your ship, spectacularly diving until they open their parachutes and land safely near the spaceport."
+
+
+
+mission "Spacediving defaulters"
+	name "Spacediving at <planet>"
+	description "Head to destination with <bunks> skydivers, then drop them from the edge of space as you land. They are wearing thermal spacesuits, durable parachutes and oxygen bottles to ensure their survival. Payment is 8000. This type of job has risks of non-payment and fines."
+	passengers 5 10
+	repeat
+	job
+	source
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	destination
+		distance 1 7
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+	to offer
+		has "Spacediving professionals: done"
+		random < 1
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		dialog "As you descend through the upper atmosphere, the skydivers open the airlocks and jump quickly to avoid paying you. They open their parachutes and land far away from the spaceport."
```

Comment:
Why wouldn't you have arranged payment in advance? This makes you the player seem like a dummy.

Also, there is no difference in the mission description to suggest that this party might not pay, so you're annoying the player for basically no reason. I wrote a similar mission that seems identical to a normal one but hurts the player (shady passengers double-cross) but for that one, the point was to emphasize the risks of associating with dangerous wanted criminals. I don't see any similar point or lesson being communicated here.

Reply:
What if I add something like "you report the non-payment to the authorities. That doesn't help you to get paid, but the skydivers might have a much harder time to get off the planet again."

Reply:
I removed the defaulters and replaced them by the accident and disaster versions.